source/hiera/3.0/release_notes.markdown

```diff
@@ -3,8 +3,17 @@ layout: default
 title: "Hiera 3: Release Notes"
 ---
 
+[puppet-agent]: https://docs.puppetlabs.com/puppet/4.2/reference/about_agent.html
 
-## Hiera 3.0.3 and 3.0.2
+## Hiera 3.0.4
+
+Released June 24, 2015.
+
+Shipped in [puppet-agent] version 1.2.7.
+
+Hiera 3.0.4 adds functionality for AIX 5.3, 6.1, and 7.1, and Solaris 11.
```

Comment:
I would drop this part as technically nothing changed in hiera to support these new platforms (it's just ruby code unlike facter).